### ISSUE TITLE
Mirror.xyz as another channel for the TEC international languages

### ISSUE THREAD

**Issue by tamarandom** [CommonsBuild/coordination#1191]
## 👇🏽  What is this issue about? Give as much detail as you can. 
Look into it.

## ✨  What does success look like? (i.e., the criteria to meet to call this issue done)
1.
2.
3.

## 🧍🏻‍♀️🧍🏻‍♂️  Who should be part of the Advice Process? (People who should be consulted about this issue. Those who will have valuable input.)
@

## 👀  Who should be in the loop about this issue?
@

**Comment by chuygarcia92** [comment]
Status update: considering setting up mirror for the international blog. 
Original articles could link to the mirror translated versions of the same article.
To be discussed in Translations TEAM and Comms sync calls next week.

**Comment by GriffGreen** [comment]
@laurenluz @iviangita :eyes:

**Comment by laurenluz** [comment]
I can look into this! I have an issue for it in giveth comms as well

**Comment by laurenluz** [comment]
Would love to be in on this discussion! I'll join the comms sync next week to discuss.

We still plan to use medium? Or are we thinking to eventually transition fully?

Wanted to look into mirror for giveth as well

**Comment by tamarandom** [comment]
Awesome @chuygarcia92 @laurenluz @iviangita. Also so you are aware, @gideonro has been digging into Mirror and talking with them about their future plans. 

From my POV mirror has a lot of potential but a total migration might be premature at the moment. I could imagine posting in both places for some period of time, giving us time to test our mirror and still keeping the audience in Medium.

**Comment by gideonro** [comment]
Yes, we should discuss. I had an informative conversation with one of their devs on Thursday.

**Comment by chuygarcia92** [comment]
As the Translations TEAM keeps moving forward, we're starting to set up a Mirror profile for international content that can offer sub categories that divide content by languages.

As a supporting action, we would kindly suggest the Medium content authors to add an edit with the available languages for the content.

**Comment by mount-manu** [comment]
**Chuy's Update:** This issue is a function of the Translations T.E.A.M., for which a Gravity case was opened recently. However, the misunderstanding has been resolved and this issue is expected to be unblocked soon as the obstacles have been removed and solutions are in the process of being applied.

**Comment by chuygarcia92** [comment]
**UPDATE**:
The gravity case has reached [a solution](https://forum.tecommons.org/t/comms-t-e-a-m-dao-funding-proposal/865/7?u=chuygarcia92). And we'll be moving forward with the mirror setup proposal on the next Translations TEAM sync, to happen on Monday, May 30th.

**Comment by chuygarcia92** [comment]
#### UPDATE: 
First translations are up for [Wielding Promethean Fire — The Importance of Token Engineering Public Goods](https://medium.com/token-engineering-commons/wielding-promethean-fire-the-importance-of-te-public-goods-9afba0ce5a43) by @jessicazartler 

**Spanish:**
https://mirror.xyz/0x6Ed3cb62B5E28F3D38C5c1E495a5666075091c7E/zOO2pC6RbiHBCDU1jeKOauP0Ji_j697ZFpZ9KGuejbM

**Turkish:**
https://mirror.xyz/0x6Ed3cb62B5E28F3D38C5c1E495a5666075091c7E/AmLXeWbMmwvsg6mk_M14gUQB6ItmK9vSiHdciFG7oIo

**Comment by natenatesoo** [comment]
@chuygarcia92 Can we close this issue?

https://mirror.xyz/0x6Ed3cb62B5E28F3D38C5c1E495a5666075091c7E

Here is the channel, and I assume this has been accomplished and any other efforts can be created as a separate issue.  

I will close this issue with a comment, and if this issue needs to be re-opened for any reason, please do so and place within the correct pipeline! Thanks!
